protoc-gen-swagger/options/openapiv2.proto

```diff
@@ -11,6 +11,28 @@ import "google/protobuf/struct.proto";
 //
 // See: https://github.com/OAI/OpenAPI-Specification/blob/3.0.0/versions/2.0.md#swaggerObject
 //
+// Example:
+//
+//  option (grpc.gateway.protoc_gen_swagger.options.openapiv2_swagger) = {
+//	  info: {
+//		  title: "Echo API";
+//		  version: "1.0";
+//      description: ";
+//		  contact: {
+//			  name: "gRPC-Gateway project";
+//			  url: "https://github.com/grpc-ecosystem/grpc-gateway";
+//			  email: "none@example.com";
+//		  };
+//		  license: {
+//			  name: "BSD 3-Clause License";
+//			  url: "https://github.com/grpc-ecosystem/grpc-gateway/blob/master/LICENSE.txt";
+//		  };
+//    };
+//	  schemes: HTTPS;
+//	  consumes: "application/json";
+//	  produces: "application/json";
+//  };
```

Comment:
Looks like this is mixing tabs and spaces, maybe best to use spaces since this is a comment? 2 spaces width is usual protobuf format I think.

Reply:
Grrr. passed it though a linter beforehand, but something must have gone wrong. Should be okay now.